requirements.txt

```diff
@@ -6,7 +6,7 @@ argcomplete
 bcrypt
 cryptography==2.4.2
 eventlet==0.24.1
-flex>=6.13.2,<6.15.0
+flex
```

Comment:
In your other PR at https://github.com/StackStorm/st2/pull/4510, `flex` is pinned to `flex>=6.13.2,<6.15.0`.  How come the version specific dependencies are gone?

Reply:
We are missing an entry for flex in ``fixed-requirements.txt``.

I will go ahead and add it.

Reply:
Should be good to go now - d29cd7baa.